packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -40,9 +40,14 @@ class Input extends Component {
         this.props.onReset();
     }
 
-    onExpandOrCollapse(e) {
+    onExpandOrCollapse(e, isSuggestionsShowing) {
```

Comment:
Can you get this value from `this.props.isSuggestionsShowing` instead of passing it as an argument?